src/mongo/MongoScrapbook.ts

```diff
@@ -294,12 +285,62 @@ class FindMongoCommandsVisitor extends MongoVisitor<MongoCommand[]> {
 				parsedObject = [];
 			}
 		} else if (child instanceof mongoParser.FunctionCallContext || child instanceof ErrorNode) {
-			return {};
+			if (child.text.startsWith("ObjectId")) {
```

Comment:
What's child.text usually?  Don't like matching without word boundaries.

Reply:
Don't rely so much on the text - you're already using a parser, so take advantage of that.  function name is child.children[0].text (just make sure to do non-null checks)